bin/reacli.js

```diff
@@ -106,7 +106,45 @@ const parseIndex = (componentName) => {
 	return data.replace(/MyComponent/gu, componentName)
 }
 
-const createComponent = (path) => {
+const applyFlowOption = (dumbString, containerString, componentName) => {
+	containerString = containerString.replace(/\[\[flow-declaration\]\]/gu, "// @flow")
+	containerString = containerString.replace(/\[\[flow-props-type\]\]/gu, "type Props = {\n\n};");
+	containerString = containerString.replace(/\[\[flow-state-type\]\]/gu, "type State = {\n\tvalue1: string,\n};");
+	containerString = containerString.replace(/\[\[flow-component-typing\]\]/gu, "<Props, State>");
+	containerString = containerString.replace(/\[\[flow-default-props-static\]\]/gu, "static defaultProps = {\n\n\t};");
+
+	dumbString = dumbString.replace(/\[\[flow-declaration\]\]/gu, "// @flow")
+	dumbString = dumbString.replace(/\[\[flow-props-type\]\]/gu, "type Props = {\n\tvalue1?: string,\n};");
+	dumbString = dumbString.replace(/\[\[flow-dumb-component-props-typing\]\]/gu, ": Props")
+	dumbString = dumbString.replace(/\[\[flow-default-props-out\]\]/gu, `${componentName}.defaultProps = {\n\tvalue1: '',\n};`);
+
+	return [
+		dumbString,
+		containerString,
+	]
+}
+
+const dismissFlowOption = (dumbString, containerString) => {
```

Comment:
I think it could be interesting to create an array or object that maps regex with the value it should be replaced with and loop over that array ;-)

Reply:
This should be ok now 👍